asyncTestPassedHead.js

```diff
@@ -0,0 +1,16 @@
+/* jshint esversion: 6, unused: false, undef: false */
```

Comment:
why do we need these extra files littering the root of the project?

Reply:
> why do we need these extra files littering the root of the project?
We don't need to add extra files, I added their content to jerryscript.js.